.github/workflows/pull-request-with-code.yml

```diff
@@ -18,30 +18,28 @@ on:
   workflow_dispatch:
 
 env:
-  ORG_GRADLE_PROJECT_signingKey : ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
+  ORG_GRADLE_PROJECT_signingKey: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
   ORG_GRADLE_PROJECT_signingKeyId: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEYID }}
-  ORG_GRADLE_PROJECT_signingKeyPassword : ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
+  ORG_GRADLE_PROJECT_signingKeyPassword: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
   CLI_TEST_MAX_DURATION_IN_SECONDS: 10
 
 # Note that "jobs.build.strategy" and "jobs.build.runs-on" should be kept in sync with "pull-request-wth-code"
 jobs:
   build:
     strategy:
       matrix:
-        os: [ ubuntu-latest, windows-latest ]
-        # When changing the list of JDK versions, the build configuration has to be changed by a repository admin. See
```

Comment:
I removed this comment since it won't be relevant anymore (excluding this single time when someone has to changed via settings). If you _prefer_ having seprate checks for each java version, I can create seprate matrix that runs single task: `./gradlew testJdk${{ matrix.jdk }}`.

Reply:
My apologies, I should have re-visit this comment after I pushed CI latest updates. With my initial implementation, there was no per-JDK-version matrix, but @/Goooler pointed out an extra matrix is required because running all tests on a single GitHub node takes too much time.

>Is it correct that you offer to provide addition gradle tasks that can be used on the local machine to execute tests for a specific java version only?

Not exactly, these tasks are already there (you can checkout this branch and run `./gradlew testJdk11` to run test suite using Java 11 runtime). What I meant was to create a separate workflow step to explicitly run using specified JDK version, which I later did here: https://github.com/pinterest/ktlint/pull/2120/files#diff-0ec67fcdf5c2efde1f1460cace15d6c9c932621ecd1c5b5f2ec8116653436c13R79

I'll try to figure out a way to reduce the number of places Java version is specified, or I will re-add a comment similar to the one I removed